resources-generator/src/main/kotlin/dev/icerock/gradle/AndroidPluginLogic.kt

```diff
@@ -72,9 +73,13 @@ internal class AndroidPluginLogic(
     ): MRGenerator.SourceSet {
         return object : MRGenerator.SourceSet {
             override val name: String
-                get() = "android${androidSourceSet.name.capitalize()}"
+                get() = "android${androidSourceSet.name.replaceFirstChar { 
+                    if (it.isLowerCase()) it.titlecase(Locale.ROOT) else it.toString() }
+                }"
 
             override fun addSourceDir(directory: File) {
+                androidSourceSet.kotlin.srcDirs(directory)
+                androidSourceSet.java.srcDirs(directory)
                 kotlinSourceSets.forEach { it.kotlin.srcDir(directory) }
```

Comment:
looks expensive. it's not create duplication?

Reply:
removed all exclude java.srcDirs